migrations/20200804025157-create-models.js

```diff
@@ -0,0 +1,311 @@
+      )
+      ;
+      -- 
+      -- TABLE: emergencymedicalservicecall 
+      --
+      CREATE TABLE emergencymedicalservicecall(
+          emergencymedicalservicecall_uuid    uuid           NOT NULL,
+          dispatchcallnumber                  int4           NOT NULL,
+          startdatetime                       timestamp      NOT NULL,
+          recordcreatetimestamp               timestamp      NOT NULL,
+          recordcreatesource                  varchar(50)    NOT NULL,
+          recordupdatetimestamp               timestamp      NOT NULL,
+          recordupdatesource                  varchar(50)    NOT NULL
+      )
+      ;
+      -- 
+      -- TABLE: emergencymedicalserviceprovider 
+      --
+      CREATE TABLE emergencymedicalserviceprovider(
+          emergencymedicalserviceprovider_uuid    uuid           NOT NULL,
+          emergencymedicalserviceprovidername     varchar(40)    NOT NULL,
+          recordcreatetimestamp                   timestamp      NOT NULL,
+          recordcreatesource                      varchar(50)    NOT NULL,
+          recordupdatetimestamp                   timestamp      NOT NULL,
+          recordupdatesource                      varchar(50)    NOT NULL
+      )
+      ;
+      -- 
+      -- TABLE: hospital 
+      --
+      CREATE TABLE hospital(
+          hospital_uuid            uuid           NOT NULL,
+          hospitalname             varchar(60)    NOT NULL,
+          recordcreatetimestamp    timestamp      NOT NULL,
+          recordcreatesource       varchar(50)    NOT NULL,
+          recordupdatetimestamp    timestamp      NOT NULL,
+          recordupdatesource       varchar(50)    NOT NULL
+      )
+      ;
+      -- 
+      -- TABLE: hospitaladministrator 
+      --
+      CREATE TABLE hospitaladministrator(
+          hospitaladministrator_uuid         uuid           NOT NULL,
+          hospital_uuid                      uuid           NOT NULL,
+          hospitaladministratoridentifier    varchar(50)    NOT NULL,
+          firstname                          varchar(50)    NOT NULL,
+          lastname                           varchar(50)    NOT NULL,
+          recordcreatetimestamp              timestamp      NOT NULL,
+          recordcreatesource                 varchar(50)    NOT NULL,
+          recordupdatetimestamp              timestamp      NOT NULL,
+          recordupdatesource                 varchar(50)    NOT NULL
+      )
+      ;
+      -- 
+      -- TABLE: hospitalstatusupdate 
+      --
+      CREATE TABLE hospitalstatusupdate(
+          hospitalstatusupdate_uuid             uuid             NOT NULL,
+          hospital_uuid                         uuid             NOT NULL,
+          updatedatetime                        timestamp        NOT NULL,
+          hospitaladministrator_uuid            uuid             NOT NULL,
+          openedbedcount                        int2,
```

Comment:
@billhoke Should `openedbedcount` just be `openbedcount`? Does this column represent a delta or the current total?

Reply:
Upon further examination, I determined that the logical name upon which this column name is based was not Opened Bed Count. Rather, it is Open ED Bed Count. It's confusing but not a mistake. Leaving as-is for now, but open to ideas ... I do think we need to be specific in case variations of bed count turn out to be required. (It seems to me like a topic still needing vetting by ED reviews ... many of which are still ahead of us.)